pkg/distributor/distributor.go

```diff
@@ -187,7 +189,8 @@ type Config struct {
 	// These functions will only receive samples that don't get dropped by HA deduplication.
 	PushWrappers []PushWrapper `yaml:"-"`
 
-	WriteRequestsBufferPoolingEnabled bool `yaml:"write_requests_buffer_pooling_enabled" category:"experimental"`
+	WriteRequestsBufferPoolingEnabled           bool `yaml:"write_requests_buffer_pooling_enabled" category:"experimental"`
+	LimitInflightRequestsUsingGrpcMethodLimiter bool `yaml:"limit_inflight_requests_using_grpc_method_limiter" category:"experimental"`
```

Comment:
Looks like it's not used. I guess it's just one of those piece not implemented yet.

Reply:
It's used from here: https://github.com/grafana/mimir/blob/bc26fa26a24f33e00d7e546032904294569dfa13/pkg/mimir/modules.go#L291